src/work-allocation/components/constants/config.constants.ts

```diff
@@ -63,6 +77,13 @@ const DUE_DATE: TaskFieldConfig = {
   sortName: 'dueDate',
   views: TaskView.ALL_VIEWS
 };
+const AGE_OF_TASK: TaskFieldConfig = {
+  name: 'dueDate',
```

Comment:
`dueDate` vs `AGE_OF_TASK`? This doesn't look right...

Reply:
the name of the field has now changed to date